### ISSUE TITLE
`metsHdr/agent/name` is not 0..*, but 1..1

### ISSUE THREAD

**Issue by jmaferreira** [DILCISBoard/E-ARK-SIP#101]
SIP12 `metsHdr/agent/name` is defined as 0..* MAY, but METS defines it as 1..1 (mets.xsd, lines 247-263):

```xml
<xsd:element name="metsHdr" minOccurs="0">
  <xsd:complexType>
    <xsd:sequence>
      <xsd:element name="agent" minOccurs="0" maxOccurs="unbounded">
        <xsd:complexType>
          <xsd:sequence>
            <xsd:element name="name" type="xsd:string">
```

Duplicates #102

**Comment by jmaferreira** [comment]
<img width="706" alt="Screenshot 2022-05-03 at 08 13 17" src="https://user-images.githubusercontent.com/3181302/166416376-0226e3da-3119-4be6-94fd-6bb473a3e63b.png">

**Comment by jmaferreira** [comment]
A fix has been issued on commit 432e860614984d3cd0bd1e7a2710f05b0bcd4a1c. Waiting for a Pull Request to merge it into master.

**Comment by jmaferreira** [comment]
Resolved.
